### PR TITLE
Revamp how tab extensions are loaded

### PR DESCRIPTION
**Pull request by brionmario**
### Purpose

Some unit test specs were bundled into the production bundle. The way we currently load tab extensions could be a potential suspect.

### Related Issues
- N/A

### Related PRs
- N/A

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: apps/console/src/features/identity-providers/components/edit-multi-factor-authenticator.tsx [hunk 1]
```diff
@@ -19,19 +19,15 @@
 import { IdentityAppsApiException } from "@wso2is/core/exceptions";
 import { AlertLevels, LoadableComponentInterface, TestableComponentInterface } from "@wso2is/core/models";
 import { addAlert } from "@wso2is/core/store";
-import { ContentLoader, EmphasizedSegment, ResourceTab } from "@wso2is/react-components";
+import { ContentLoader, EmphasizedSegment, ResourceTab, ResourceTabPanePropsInterface } from "@wso2is/react-components";
```
Comment (SujanSanjula96) [inline apps/console/src/features/identity-providers/components/edit-multi-factor-authenticator.tsx:22]:
```suggestion
import { ContentLoader, EmphasizedSegment, ResourceTab, ResourceTabPaneInterface } from "@wso2is/react-components";
```

File: apps/console/src/features/identity-providers/components/edit-multi-factor-authenticator.tsx [hunk 2]
```diff
@@ -116,15 +112,10 @@ export const EditMultiFactorAuthenticator: FunctionComponent<EditMultiFactorAuth
             return;
         }
 
-        // Resolve the quick-start tab.
-        const extensions: any[] = ComponentExtensionPlaceholder({
-            component: "identityProvider",
-            props: {
+        const extensions: ResourceTabPanePropsInterface[] = identityProviderConfig
```
Comment (SujanSanjula96) [inline apps/console/src/features/identity-providers/components/edit-multi-factor-authenticator.tsx:115]:
```suggestion
        const extensions: ResourceTabPaneInterface[] = identityProviderConfig
```

Comment (brionmario) [reply apps/console/src/features/identity-providers/components/edit-multi-factor-authenticator.tsx]: Addressed with https://github.com/wso2/identity-apps/pull/3609/commits/aacb22bac0bba67f081f7730b5def72f74aa77cb

File: apps/console/src/features/identity-providers/components/edit-multi-factor-authenticator.tsx (hunk 1 above)
Comment (brionmario) [reply apps/console/src/features/identity-providers/components/edit-multi-factor-authenticator.tsx]: Addressed with https://github.com/wso2/identity-apps/pull/3609/commits/aacb22bac0bba67f081f7730b5def72f74aa77cb

**Review by SujanSanjula96 — APPROVED** [review]

**Pull request merged**